test/e2e/common/configmap.go

```diff
@@ -155,7 +155,7 @@ var _ = framework.KubeDescribe("ConfigMap", func() {
 
 	It("should be consumable via environment variable [Conformance]", func() {
```

Comment:
This should be a new test.  Keep the old test around and add a new one for `EnvFrom`